checkstyle-tester/README.md

```diff
@@ -219,11 +225,19 @@ First you must build sevntu checks:
 ```
 cd sevntu-checks
 mvn  -Pno-validations clean install
-``` 
+```
 Sevntu's current version must be referenced at https://github.com/checkstyle/contribution/blob/master/checkstyle-tester/pom.xml#L16 .
 
 Finally, just run checkstyle-tester as described above.
 
+## Abstract Syntax Tree Regression Report
+
+When making changes to Checkstyle's grammar, it is important to verify that there is no regression of the AST.  This is accomplished by using the `launch_diff_antlr.sh` script, and selecting one project at a time from `projects-to-test-on.properties`.  To use this script, first you must modify the `launch_diff_variables.sh` file to reflect the location of the variables that `launch_diff_antlr.sh` uses to produce the report. Please note that `PULL_REMOTE` will need to be set to the name of the remote repository where your branch resides. If your branch is found at `origin/name-of-your-branch-here`, you will set
```

Comment:
> and selecting one project at a time from `projects-to-test-on.properties`

Unless denoted otherwise, I believe Checkstyle should always be the 1 project. We pull in so many different sources from other places that we are unlike any other project.

Reply:
Fixed.